app_dart/lib/src/request_handlers/update_existing_flaky_issues.dart

```diff
@@ -79,4 +90,48 @@ class UpdateExistingFlakyIssue extends ApiRequestHandler<Body> {
       }
     }
   }
+
+  /// Updates existing flaky issues based on corrresponding builder stats.
+  Future<void> _updateExistingFlakyIssue(
+    GithubService gitHub,
+    RepositorySlug slug,
+    pb.SchedulerConfig schedulerConfig, {
+    required List<BuilderStatistic> prodBuilderStatisticList,
+    required List<BuilderStatistic> stagingBuilderStatisticList,
+    required Map<String?, Issue> nameToExistingIssue,
+  }) async {
+    final Map<String, bool> builderFlakyMap = <String, bool>{};
+    for (pb.Target target in schedulerConfig.targets) {
+      builderFlakyMap[target.name] = target.bringup;
+    }
+    // For prod builder stats, updates an existing flaky bug when the builder is not marked as flaky.
```

Comment:
do we need to worry about cases where a test is recently move to a staging, there are recent builds both in prod and stage

Reply:
If that is the case, would that be noisy? for example, a test recently moved from `prod` to `stage` and there is only 1 `stage` while other existing builds are in prod. wouldn't we want to grab at least 100 recent builds for the stats?

Reply:
That's a good point. I added a threshold (10 commits) to control if we want to update the flaky ratio. Does that sound good to you?

Reply:
sounds good